dgp/utils/dataset_conversion.py

```diff
@@ -68,21 +68,22 @@ def compute_image_statistics(image_list, image_open_fn):
         num_processes = len(image_list)
 
     chunk_size = int(len(image_list) / num_processes)
-    p = Pool(num_processes)
 
-    image_stats_per_process = p.starmap(
-        _get_image_stats,
-        [(image_list[i:i + chunk_size], i, num_processes, image_open_fn) for i in range(0, len(image_list), chunk_size)]
-    )
+    with Pool(num_processes) as p:
```

Comment:
Seems like `Pool` is from python `multiprocessing` -- this can conflict with `torch.multiprocessing` depending on what else is loaded in environment. 

According to pytorch code comments, the base package doesn't fully clean up after itself (and I have seen this behavior in learned track development) 
https://github.com/pytorch/pytorch/blob/master/torch/multiprocessing/pool.py#L10